saleor/static/scss/dashboard/layouts/_style-guide.scss

```diff
@@ -0,0 +1,102 @@
+.styleguide {
+  &__menu {
+    float: right !important;
```

Comment:
Maybe it'll be better to use push and pull from materialize grid to replace columns order in small devices?